tests/integration_test/services_test.py

```
@@ -6904,6 +6904,312 @@
+        sync_type="push_menu",
+        start_timestamp=datetime.utcnow(),
+        sync_status="Initiated",
+        status="In Progress"
+    ).save()
+
+    push_menu_payload_path = Path("tests/testing_data/catalog_sync/catalog_sync_push_menu_payload.json")
+
+    with push_menu_payload_path.open("r", encoding="utf-8") as f:
+        push_menu_payload = json.load(f)
+
+    response  = client.post(
```

Flake8 (7.2.0)

[error] multiple spaces before operator

(E221)